config/locales/views/onboarding/onboarding.yml

```diff
@@ -171,6 +171,8 @@ en:
         title: Edit account
         update_account: Update account
       form:
+        preferred_locale: Preferred language
+        preferred_locale_hint: This will be the language website and email content will be in for this user
```

Comment:
"Set your preferred language for using the website and receiving emails"

Reply:
:+1: done